### PR TITLE
Sec forms

### PR DESCRIPTION
**Pull request by ghost**
Also a test to try to remember how git works and to see if I completely broke anything.

Adds new SEC forms:

- arrest_report.txt

- hot_sheet.txt

- patrol_assignment2.txt

- personnel_watchlist2.txt

- secarm_inventory.txt

- shift_report.txt

### CODE REVIEW

**Comment by soryy708** [comment]
Looks like you did great, not breaking anything git-wise.
Can you fix the merge conflict?

**Comment by Dibasic** [comment]
I'll review in a bit, going to adjust the JSON formatting but the forms themselves should be fine.

If your competing patrol and armory forms look good I'll merge them in, but I want to have one best version of each form, so I'll open issues about combining them or picking a best template to go with.

Not sure why Codebeat thinks this is introducing 900 errors.

**Comment by Dibasic** [comment]
Also, you don't have to squash every commit you make, GitHub now lets the maintainer squash commits when merging.

File: templates/index.json [hunk 1]
```diff
@@ -12,9 +12,15 @@
...
+        "Arrest Report": "security/arrest_report.txt",
+		"Hot Sheet": "security/hot_sheet.txt",
+		"Incident Report": "security/incident_report.txt",
         "Patrol Assignment": "security/patrol_assignment.txt",
-        "POI Watchlist": "security/personnel_watchlist.txt"
+		"Patrol Assignment (Alternate)": "security/patrol_assignment2.txt",
+        "POI Watchlist": "security/personnel_watchlist.txt",
+		"POI Watchlist (Alternate)": "security/personnel_watchlist2.txt",
+		"SECARM Inventory": "security/secarm_inventory.txt",
+		"POI Watchlist": "security/shift_report.txt"
```
Comment (Dibasic) [inline templates/index.json]:
```suggestion
		"Shift Report": "security/shift_report.txt"
```

**Review by Dibasic — CHANGES_REQUESTED** [review]

**Pull request closed without merging**